namespaces/live.cloud-platform.service.justice.gov.uk/laa-crime-application-store-uat/resources/rds-postgresql7.tf

```diff
@@ -0,0 +1,73 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "rds_metabase" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=6.0.2"
+
+  # VPC configuration
+  vpc_name = var.vpc_name
+
+  # RDS configuration
+  allow_minor_version_upgrade  = true
+  allow_major_version_upgrade  = false
+  performance_insights_enabled = false
+  db_max_allocated_storage     = "500"
+  deletion_protection          = true
+  # enable_rds_auto_start_stop   = true # Uncomment to turn off your database overnight between 10PM and 6AM UTC / 11PM and 7AM BST.
```

Comment:
@imtiazAhmd I think we were saying in slack chat the we would `enable_rds_auto_start_stop` on UAT. I have already for app store UAT main database